File: DSA/Assignment_1/Q8.java
/*Name-Anish More
 Registration Number-2241013365
 Name Of the program-Write a java program to print M-by-N array in the tabular format. Also, display the sum of
elements of the array. */
import java.util.Scanner;
public class Q8 
{
	public static void main(String[] args) 
	{
		Scanner sc=new Scanner(System.in);
		System.out.print("Enter number of Row and Columns of 2D-Array: ");
		int r=sc.nextInt();
		int c=sc.nextInt();
		System.out.print("Enter elements of 2D-Array: ");
		int[][] arr=new int[r][c];
		int sum=0;
		for(int i=0;i<r;i++)
		{
			for(int j=0;j<c;j++)
			{
			 arr[i][j]=sc.nextInt();
			}
		}
		for(int i=0;i<r;i++)
		{
			for(int j=0;j<c;j++)
			{
			 System.out.print(arr[i][j]+" ");
			 sum+=arr[i][j];
			}
			System.out.println();
		}
		System.out.println("The sum of elements of the 2D-Array is "+sum);
	}
}
/*Output---
Enter number of Row and Columns of 2D-Array: 3 3
Enter elements of 2D-Array: 1 2 3 2 3 4 3 4 5
1 2 3 
2 3 4 
3 4 5 
The sum of elements of the 2D-Array is 27
*/

File: PPWC/Minor_Assignment_4/Q8.c
#include <stdio.h>

int main(void){
    int arr[10] = {0, 10, 20,30, 40, 50,60, 70, 80,90};
    int size = sizeof(arr) / sizeof(arr[0]);
    for (int i = 0; i < size; i++){
        printf("%u -> %d\n", &arr[i], arr[i]);
    }
}

/*
Output--
6422256 -> 0
6422260 -> 10
6422264 -> 20
6422268 -> 30
6422272 -> 40
6422276 -> 50
6422280 -> 60
6422284 -> 70
6422288 -> 80
6422292 -> 90
*/